src/NuGet.Core/NuGet.Protocol.Core.v3/Model/PackageSearchMetadata.cs

```diff
@@ -17,7 +18,16 @@ public class PackageSearchMetadata : IPackageSearchMetadata
         public string Authors { get; private set; }
 
         [JsonProperty(PropertyName = JsonProperties.DependencyGroups, ItemConverterType = typeof(PackageDependencyGroupConverter))]
```

Comment:
Facilitates clear properties mapping of wire protocol to metadata object. Allows finer control over deserialization process.